frontend/App.tsx

```diff
@@ -15,6 +15,16 @@ export function App(props) {
     <QueryClientProvider client={queryClient}>
       <head>
         <link rel="stylesheet" href="http://localhost:7700/main.css"></link>
+        <link rel="preconnect" href="https://fonts.googleapis.com"></link>
```

Comment:
This seems unnecessary to me since we already have fonts.googleapis.com later on on line 21. The one for fonts.gstatic.com makes sense though as that one isn't connected to elsewhere and will be a result of the source sans pro font. I'm guessing you copied this from the google fonts website? If so, it's fine to keep it - doesn't hurt

Reply:
Ah, interesting.  Yes, it was a direct copy from google fonts website.